src/Downloaders/DefaultDownloader.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Spatie\MediaLibrary\Downloaders;
+
+use Spatie\MediaLibrary\MediaCollections\Exceptions\UnreachableUrl;
+
+class DefaultDownloader
```

Comment:
Let's make this class implement an `Downloader` interface with that `getTempFile` method in it. In the readme and config file it should be mentioned. That a valid downloader is any class that implements that new interface.